backend/routes/index.js

```diff
@@ -38,4 +46,28 @@ router.get( '/calendar/:calendar', async ( req, res ) => {
    }
 } );
 
+router.put( '/calendar/:calendar', async ( req, res ) => {
```

Comment:
By convention http put should be used to update an existing item. Http post method is more suitable here unless we intrerpret this action as "update a calendar" instead of "create an event".

Reply:
Fixed in #46 .